tokens/sys/brand/default.json

```diff
@@ -132,15 +132,15 @@
     },
     "common": {
       "focus-outline": {
-        "value": "{brand.primary.base}",
+        "value": "{palette.blueberry.400}",
         "type": "color"
       },
       "error-inner": {
-        "value": "{brand.error.base}",
+        "value": "{palette.cinnamon.500}",
         "type": "color"
       },
       "alert-inner": {
-        "value": "{brand.alert.base}",
+        "value": "{palette.cantaloupe.400}",
```

Comment:
Q: Why revert these back to base tokens instead of their brand token equivalents?

Reply:
Don't reference values in the same file – resolved.